saleor/graphql/order/mutations/orders.py

```diff
@@ -781,6 +785,8 @@ def perform_mutation(cls, _root, info, **data):
         )
 
         recalculate_order(order)
+        transaction.on_commit(lambda: info.context.plugins.order_updated(order))
```

Comment:
I believe that this mutation can be also used for draft order, which will raise an incorrect webhook event

Reply:
changed to raise webhook event only for order lines other than draft